classes/Infrastructure/Event/TwigGlobalsListener.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
     public static function getSubscribedEvents()
     {
         return [
-            KernelEvents::REQUEST => ['onKernelRequest', 512],
+            KernelEvents::REQUEST => ['onKernelRequest', -1024],
```

Comment:
This might be too low. If for instance the router stops the event propagation (because of a 404), the exception listener would not get the globals when rendering the error page.

Reply:
What would be a good priority for this listener?

Reply:
The router should be on 32, so anything between 32 and 512 should do. 256 maybe?